src/components/profile/UserProfile.js

```diff
@@ -49,10 +51,26 @@ class UserProfile extends Component {
             )
         });
 
-        let ratedQualities = UserData[this.props.userID].qualities.map((data) => {
+       
+    let allTags = UserData[this.props.userID].reviews
```

Comment:
No need to change anything for now, but just a heads-up that this, along with almost everything that uses UserData in this component, will break when UserData will be populated by an API callback. Initially UserData will be null causing this page to break, but then when the callback returns, it will work. I'm not sure what the correct course of action is. Do you have any ideas?

Reply:
We could probably use optional chaining for it. But when we connect, it's fine for now.